lib/src/ui/issue_form.dart

```diff
@@ -30,17 +30,16 @@ class _IssueFormState extends State<IssueForm> {
     setState(() => isLoading = true);
     try {
       final template = widget.templates.firstWhere(
-        (template) => template.name == templateType,
+        (template) => template.title == templateType,
         orElse: () => IssueTemplate(
-          name: '',
-          downloadUrl: '',
+          title: '',
+          content: '',
         ),
       );
-      final templateBody =
-          await widget.service!.fetchTemplateBody(template.downloadUrl);
```

Comment:
Good that you removed the call `fetchTemplateBody`, you should also remove that `fetchTemplateBody` function from the service file too. Since we no longer need it @Oyebliss